content/sensu-go/6.2/operations/deploy-sensu/datastore.md

```diff
@@ -248,11 +251,22 @@ default     | false
 type         | Boolean
 example      | {{< code shell >}}strict: true{{< /code >}}
 
+<a name="round-robin-postgresql"></a>
+
+enable_round_robin |      |
+-------------|------
+description  | If `true`, enables [round robin scheduling][5] on PostgreSQL. Any existing round robin scheduling will stop and migrate to PostgreSQL as entities check in with keepalives. Sensu will gradually delete the existing etcd scheduler state as keepalives on the etcd scheduler keys expire over time. Otherwise, `false`.
```

Comment:
Seems like we are missing some context as to why a person would want to turn this on. If it's not approrpriate for this section, perhaps we need to describe the options in `observability-pipeline/observe-schedule/checks/#round-robin-checks` ?

Reply:
Thank you. I added some details here and in the checks reference. There may be other helpful details in https://github.com/sensu/sensu-enterprise-go/issues/1345 but I'm not sure.